.github/workflows/next-container-build.yaml

```
@@ -12,11 +12,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-name: Build operator, bundle, and catalog images
+# for main branch, use next tags; for 1.x branches, use :latest tags
+name: Build and push operator, bundle, and catalog images
 
 on:
   push:
-    branches: [ main ]
+    branches: 
+    - main
+    - rhdh-1.**
+    - 1.**.x
```

`**` seems to match the '/' character, which I don't think we want: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet
Also, shouldn't we be more specific about the minor number? We may not want to match branches like `1.foo/bar.x`, no?

```suggestion
    - 1.[0-9]+.x
```